dashboard/src/components/InfoCard/InfoCard.v2.tsx

```diff
@@ -0,0 +1,84 @@
+import { ClarityIcons, infoCircleIcon } from "@clr/core/icon-shapes";
+import * as React from "react";
+import { Link } from "react-router-dom";
+import { CdsIcon } from "../Clarity/clarity";
+
+import placeholder from "../../placeholder.png";
+import "./InfoCard.v2.css";
+
+ClarityIcons.addIcons(infoCircleIcon);
+
+export interface IInfoCardProps {
+  title: string;
+  info: string | JSX.Element;
+  link?: string;
+  icon?: string;
+  banner?: string;
+  subIcon?: string;
+  description?: string | JSX.Element;
+  tag1Class?: string;
+  tag1Content?: string | JSX.Element;
+  tag2Class?: string;
+  tag2Content?: string | JSX.Element;
+}
+
+function InfoCard(props: IInfoCardProps) {
+  const {
+    title,
+    link,
+    info,
+    description,
+    tag1Content,
+    tag1Class,
+    tag2Content,
+    tag2Class,
+    banner,
+    subIcon,
+  } = props;
+  const icon = props.icon ? props.icon : placeholder;
+  return (
+    <div className="clr-col-lg-3 clr-col-12">
+      <Link to={link || "#"} className="card clickable">
+        <div className="card-header">{title}</div>
+        <div className="card-block">
```

Comment:
You can use the internal `Card` component. It contains several subcomponents such as `CardHeader` and `CardBlock`.